[docs-agent] Deprecate Account Kit (v4) in wallets nav (#1530)
alchemyplatform/docs: content/wallets/pages/account-kit-deprecated.mdx

---
title: "[Deprecated] Account Kit"
description: Account Kit (v4) is deprecated. Wallet APIs is the supported successor.
slug: wallets/account-kit-deprecated
---

> Account Kit, including embedded wallets, signer support, and key management (React SDK, embedded UI components, login methods, etc) is deprecated and its docs have been retired.

Reviewer: ```suggestion
Account Kit, including embedded wallets, signer support, and key management (React SDK, embedded UI components, login methods, etc) is deprecated. 
```

> Use [Wallet APIs](/docs/wallets) for all new integrations. Wallet APIs are signer agnostic and allow for sponsored transactions. 

Reviewer: ```suggestion
Use [Wallet APIs](/docs/wallets) for all new integrations. Wallet APIs are signer agnostic and allow for sponsored transactions. We recommend using Privy or another provider for signer support alongside Wallet APIs.
```

---
title: "[Deprecated] Account Kit"
description: Account Kit (v4) is deprecated. Wallet APIs is the supported successor.
slug: wallets/account-kit-deprecated
---

> Account Kit (v4), including its React SDK, embedded UI components, login methods, MFA flows, and legacy Signer quickstart, is deprecated and its docs have been retired.

Reviewer: ```suggestion
Account Kit, including all embedded wallet, signer, and key management support (React SDK, embedded UI components, login methods, etc.) is deprecated. We recommend using Privy or another Signer provider for signing alongside our Wallet APIs.
```

> Use [Wallet APIs](/docs/wallets) for all new integrations. It covers everything Account Kit did, with a smaller footprint and better support across chains.

Reviewer: ```suggestion
Use [Wallet APIs](/docs/wallets) for all new integrations. Wallet APIs are signer agnostic and enable sponsored transactions.
```

> ## Where things moved

Reviewer: ```suggestion
```

> * Get started: [Wallet APIs quickstart](/docs/wallets/quickstart)

Reviewer: ```suggestion
```

> * Authentication and login methods: [Wallet integration overview](/docs/wallets)

Reviewer: ```suggestion
```

> * External wallets and connectors: [Wallet integration](/docs/wallets)

Reviewer: ```suggestion
```

* Solana wallet flows: [Solana signers](/docs/wallets/solana/signers)
* API reference: [Wallet APIs endpoints](/docs/wallets/smart-wallets)

> The [Export private key](/docs/wallets/signer/export-private-key) page remains available for existing integrations.

Reviewer: ```suggestion
```

## Support

> For migration questions, reach out in [Alchemy's developer channels](https://www.alchemy.com/support) or open a ticket from the [dashboard](https://dashboard.alchemy.com/).

Reviewer: ```suggestion
See [migration](/docs/wallets/wallet-integrations/privy/signer-migration-overview) docs if you need to migrate old Alchemy Signer/Account Kit users.

The [Export private key](/docs/wallets/signer/export-private-key) page remains available for ongoing migrations.
```
